.ruby-version

```diff
@@ -0,0 +1 @@
+2.6.5
```

Comment:
Is the hyphen file best practice?

Reply:
`.ruby_version` isn't picked up by `rbenv` only `.ruby-version` is. so the file name change was necessary for the configuration to take affect.